SubjectsDAL/Repos/AndroidRepo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using SubjectsDAL.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SubjectsDAL.Repos
+{
+    public class AndroidRepo: BaseRepo<Android>, IRepo<Android>
```

Comment:
Don't use a shortcut like `repo` you should consider renaming this to its full name like `repository`. The same for the directory whether this file belongs to.